README.md

```diff
@@ -1,21 +1,63 @@
-# Fether
+# Parity Fether a fast and decentralized wallet based on a light-client
 
 ## Get the latest binary
 
-https://github.com/paritytech/fether/releases
+[» Download the latest release «](https://github.com/paritytech/fether/releases)
 
-## Getting started
+
+### Join the chat!
+
+Get in touch with us on Gitter:
+[![Gitter: Parity](https://img.shields.io/badge/gitter-parity-4AB495.svg)](https://gitter.im/paritytech/parity)
+
+
+Official website: https://parity.io | Be sure to check out [our Wiki](https://wiki.parity.io) for more information.
+
+----
+
+## About Parity Fether
+
+Fether aims to be the lightest and simplest decentralized wallet. It supports Ether and ERC-20 tokens and runs on top of [Parity Ethereum](https://github.com/paritytech/parity) light client allowing smooth synchronization and interaction with the Ethereum blockchain, in a decentralized manner.
+
+By default, Parity Fether alpha runs on the Kovan test network. You can receive free Kovan Ether by posting your address in the [Kovan Faucet](https://gitter.im/kovan-testnet/faucet) Gitter channel. Fether will download and launch Parity Ethereum node at startup if it's not found on the computer. You can also separately launch your Ethereum client, Fether will automatically connect to it.
+
+If you run into problems while using Parity Fether, feel free to file an issue in this repository or hop on our [Gitter](https://gitter.im/paritytech/parity) or [Riot](https://riot.im/app/#/group/+parity:matrix.parity.io) chat room to ask a question. We are glad to help! **For security-critical issues**, please refer to the security policy outlined in [SECURITY.md](https://github.com/paritytech/parity/blob/master/SECURITY.md).
+
+Parity Fether connects to the light node using light.js, a Javascript library specifically crafted for Wallets to connect with light clients. It is licensed under the BSD 3-Clause, and can be used for all your Ethereum needs.
+
+----
+
+## Start Parity Fether
+
+### Manually
+
+To start Parity Fether manually, just run
 
 ```bash
-git clone https://github.com/paritytech/fether
-yarn install
+$ fether
 ```
+You can pass specific flags to the underlying Parity Ethereum node such as:
```

Comment:
FWIW this only holds if there is no already running node (i.e. if fether starts the node)

If Parity is already running on kovan, running `fether --chain mainnet` will still connect to Parity on Kovan. (Could be good in Fether to warn the user if the user passed some flags but there is an already running Parity instance and as such the flags are ignored cc @amaurymartiny)

Reply:
+1 I will try to rephrase that to make it clear